biodatasets/bionlp_st_2013_pc/bionlp_st_2013_pc.py

```diff
@@ -0,0 +1,234 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pathlib import Path
+from typing import List
+
+import datasets
+
+from utils import parsing, schemas
+from utils.configs import BigBioConfig
+from utils.constants import Tasks
+
+_DATASETNAME = "bionlp_st_2013_pc"
+_UNIFIED_VIEW_NAME = "bigscience"
```

Comment:
```suggestion
_UNIFIED_VIEW_NAME = "bigbio"
```
_UNIFIED_VIEW_NAME should match up the name of the existing config -either source or bigbio: see example/mlee.py